bin/reset-local-e2e-tests-block-editor.js

```diff
@@ -0,0 +1,41 @@
+// Do this as the first thing so that any code reading it knows the right env.
+process.env.BABEL_ENV = 'test';
+process.env.NODE_ENV = 'test';
+process.env.JEST_PUPPETEER_CONFIG = 'packages/scripts/config/puppeteer.config.js';
+
+// Makes the script crash on unhandled rejections instead of silently
+// ignoring them. In the future, promise rejections that are not handled will
+// terminate the Node.js process with a non-zero exit code.
+process.on( 'unhandledRejection', ( err ) => {
+	throw err;
+} );
+
+/**
+ * External dependencies
+ */
+const Bundler = require( 'parcel-bundler' );
+const { join } = require( 'path' );
+// eslint-disable-next-line jest/no-jest-import
+const jest = require( 'jest' );
+
+const bundler = new Bundler(
+	join( __dirname, '../playground/src/index.html' ),
+	{
+		outDir: '../playground/dist',
+	}
+);
+
+bundler.on( 'buildEnd', () => {
+	jest.run( [
+		'--config',
+		'packages/e2e-tests-block-editor/jest.config.js',
+		'--runInBand',
+		...process.argv.slice( 2 ),
+	] ).then( () => {
+		process.exit( 0 );
+	} ).catch( () => {
+		process.exit( 1 );
+	} );
+} );
+
+bundler.serve();
```

Comment:
Is this separate script needed? I wonder if we could be able to do something like `npm run playground:start && npm run test:e2e-playground` or something like that.

The reason being to decouple the environment from running the tests.

Reply:
I tried and failed. See https://github.com/parcel-bundler/parcel/issues/1131#issuecomment-378598623. This seems to be the only way to run something after parcel is done building and is serving.